frontend/src/component/personalDashboard/ConnectSDK.tsx

```diff
@@ -56,10 +56,13 @@ export const ConnectSDK: FC<{ project: string }> = ({ project }) => {
                 <NeutralCircleContainer>2</NeutralCircleContainer>
                 Connect an SDK
             </TitleContainer>
-            <Typography>
-                Your project is not yet connected to any SDK. In order to start
-                using your feature flag connect an SDK to the project.
-            </Typography>
+            <div>
+                <div>Your project is not yet connected to any SDK.</div>
+                <div>
+                    In order to start using your feature flag connect an SDK to
+                    the project.
+                </div>
+            </div>
```

Comment:
What's going on here? Again, if you want separate paragraphs, you can use multiple `p` tags.